docs/content/concepts/sui-move-concepts/packages/custom-policies.mdx

```diff
@@ -633,7 +607,7 @@ tx.transferObjects([tuesdayUpgradeCap], tx.pure(sender));
 And finally, execute that transaction and display its effects to the console. The following snippet assumes that you're running your examples against a local network. Replace all `localnetConnection` references with `devnetConnection`, `testnetConnection`, or `mainnetConnection` to run on Devnet, Testnet, or Mainnet respectively:
 
 ```js
-import { JsonRpcProvider, RawSigner, localnetConnection }
+import { JsonRpcProvider, RawSigner, localnetConnection };
```

Comment:
This is missing the package that these things are being imported from:

```suggestion
import { JsonRpcProvider, RawSigner, localnetConnection } from "@mysten/sui.js";
```

But also, this made me realise that this example is out-of-date because of the refactorings to the TS SDK.  Let me share an updated copy of the full script with you, and then you will be able to cut it back up into the appropriate pieces for this doc.